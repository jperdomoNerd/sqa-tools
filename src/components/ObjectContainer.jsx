import EndpointName from './EndpointName';
import ObjectView from './ObjectView';

const ObjectContainer = ({ endpointUrl }) => {
  return (
    <section className="object-container">
      <EndpointName endpointUrl={endpointUrl} />
      <ObjectView />
    </section>
  );
}

export default ObjectContainer;
