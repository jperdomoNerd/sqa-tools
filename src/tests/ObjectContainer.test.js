import renderer from 'react-test-renderer';
import ObjectContainer from '../components/ObjectContainer';

describe('ObjectContainer Component', () => {
  
  it('<OjbectContainer /> renders correctly', () => {
    const tree = renderer.create(<ObjectContainer endpointUrl='https://ðŸ˜ŽðŸ˜µðŸ¤ ' />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
