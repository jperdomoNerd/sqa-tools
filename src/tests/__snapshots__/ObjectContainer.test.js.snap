// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ObjectContainer Component <OjbectContainer /> renders correctly 1`] = `
<section
  className="object-container"
>
  <div
    className="endpoint-name"
  >
    <label
      className="endpoint-name__label"
      htmlFor="endpoint-name"
    >
      Endpoint:
    </label>
    <input
      className="endpoint-name__input"
      id="endpoint-name"
      readOnly={true}
      type="text"
      value="https://ðŸ˜ŽðŸ˜µðŸ¤ "
    />
  </div>
  <div
    className="object-view"
  />
</section>
`;
